ipc/Cargo.toml

```diff
@@ -17,6 +17,10 @@ bytes  = "0.3.0"
 miow = "0.1"
 log = "0.3"
 rand = "0.3"
+tokio-core = "0.1"
```

Comment:
You can reference `tokio-core` from `server-utils`, will simplify migrations to newer versions (less changes in dependent crates)
Same for `futures`, it's re-exported by `jsonrpc-core`